.swiftlint.yml

```diff
@@ -29,11 +29,22 @@ nesting:
 
 custom_rules:
   dont_use_print:
-    included: ".*\\.swift"
+    included:
+      - ".*\\.swift"
+    excluded:
+      - "Sources/Demo"
```

Comment:
It might be useful to lint the Demo project code, are you sure you want to exclude this directory?

Reply:
This only affects the `dont_use_print` custom rule for using `_Logger.searchSDK` instead of `print`/`NSLog`.